src/nvim/screen.c

```diff
@@ -1948,6 +1948,29 @@ static inline void provider_err_virt_text(linenr_T lnum, char *err)
   decor_add_ephemeral(lnum-1, 0, lnum-1, 0, &err_decor);
 }
 
+/// Draw the line number (empty space after wrapping).
+static inline void draw_line_number(win_T *wp, linenr_T lnum, char_u **extrap)
+{
+  long num;
+  char *fmt = "%*ld ";
+
+  if (wp->w_p_nu && !wp->w_p_rnu) {
+    // 'number' + 'norelativenumber'
+    num = (long)lnum;
+  } else {
+    // 'relativenumber', don't use negative numbers
+    num = labs((long)get_cursor_rel_lnum(wp, lnum));
+    if (num == 0 && wp->w_p_nu && wp->w_p_rnu) {
+      // 'number' + 'relativenumber'
+      num = lnum;
+      fmt = "%-*ld ";
+    }
+  }
+
+  snprintf((char *)(*extrap), sizeof(*extrap),
```

Comment:
Using `sizeof(*extrap)` does not work.

Reply:
You may need to pass `sizeof(extra)` as an argument.

Reply:
Thanks, that saved my time debugging this.

I think I can pass `extra` directly.